src/components/application_manager/test/commands/hmi/navi_set_video_config_request_test.cc

```diff
@@ -0,0 +1,227 @@
+#include "application_manager/hmi_interfaces.h"
+#include "application_manager/mock_hmi_interface.h"
+#include "application_manager/event_engine/event.h"
+#include "application_manager/mock_event_dispatcher.h"
+
+namespace test {
+namespace components {
+namespace commands_test {
+namespace hmi_commands_test {
+namespace navi_set_video_config_request {
+
+using ::testing::_;
+using ::testing::ReturnRef;
+namespace am = ::application_manager;
+using am::commands::MessageSharedPtr;
+using am::commands::NaviSetVideoConfigRequest;
+using am::event_engine::Event;
+
+namespace {
+const uint32_t kAppId = 1u;
+const uint32_t kHmiAppId = 13u;
+const hmi_apis::FunctionID::eType kEventID =
+    hmi_apis::FunctionID::Navigation_SetVideoConfig;
+}  // namespace
+
+typedef SharedPtr<NaviSetVideoConfigRequest> NaviSetVideoConfigRequestPtr;
+
+class NaviSetVideoConfigRequestTest
+    : public CommandRequestTest<CommandsTestMocks::kIsNice> {
+ public:
+  NaviSetVideoConfigRequestTest() {
+    mock_app_ptr_ = CreateMockApp();
+    ON_CALL(app_mngr_, hmi_interfaces())
+        .WillByDefault(ReturnRef(mock_hmi_interfaces_));
+    ON_CALL(app_mngr_, application(_)).WillByDefault(Return(mock_app_ptr_));
+    ON_CALL(app_mngr_, application_by_hmi_app(_))
+        .WillByDefault(Return(mock_app_ptr_));
+    ON_CALL(app_mngr_, event_dispatcher())
+        .WillByDefault(ReturnRef(mock_event_dispatcher_));
+    ON_CALL(*mock_app_ptr_, hmi_app_id()).WillByDefault(Return(kHmiAppId));
+  }
+
+  MOCK(am::MockHmiInterfaces) mock_hmi_interfaces_;
+  MockAppPtr mock_app_ptr_;
+  MockEventDispatcher mock_event_dispatcher_;
+};
+
+TEST_F(NaviSetVideoConfigRequestTest, OnEvent_SUCCESS) {
+  MessageSharedPtr request_msg = CreateMessage();
+  (*request_msg)[am::strings::msg_params][am::strings::app_id] = kAppId;
+
+  NaviSetVideoConfigRequestPtr command =
+      CreateCommand<NaviSetVideoConfigRequest>(request_msg);
+
+  MessageSharedPtr event_msg = CreateMessage();
+  (*event_msg)[am::strings::params][am::hmi_response::code] =
+      hmi_apis::Common_Result::SUCCESS;
+  Event event(kEventID);
+  event.set_smart_object(*event_msg);
+
+  std::vector<std::string> empty;
+  EXPECT_CALL(
+      app_mngr_,
+      OnStreamingConfigured(
+          kHmiAppId, protocol_handler::ServiceType::kMobileNav, true, empty))
+      .Times(1);
+
+  command->on_event(event);
+}
+
+static bool ValidateList(std::vector<std::string>& expected,
+                         std::vector<std::string>& actual) {
+  if (expected.size() != actual.size()) {
+    return false;
+  }
+  for (unsigned int i = 0; i < expected.size(); i++) {
+    std::string& param = expected[i];
+    unsigned int j;
+    for (j = 0; j < actual.size(); j++) {
+      if (param == actual[j]) {
+        break;
+      }
+    }
+    if (j == actual.size()) {
+      // not found
+      return false;
+    }
+  }
+  return true;
+}
+
+TEST_F(NaviSetVideoConfigRequestTest, OnEvent_FAILURE) {
```

Comment:
OK, I've renamed the test cases in https://github.com/smartdevicelink/sdl_core/pull/1700/commits/63424e01d206472a49ff7703642b85ae0f2a7e26. They do not align with other commands' test case names but they should look clearer now.